src/pages/Location/SoilCondition/SoilDrainage/RenderDrainageClasses.js

```diff
@@ -15,7 +15,8 @@ const RenderDrainageClasses = ({ tilingCheck, drainage = [] }) => {
   // redux vars
   const soilDataRedux = useSelector((stateRedux) => stateRedux.soilData.soilData);
   const councilShorthandRedux = useSelector((stateRedux) => stateRedux.mapData.councilShorthand);
-  const [previousDrainage, setPreviousDrainage] = useState(0);
+  const [previousDrainage, setPreviousDrainage] = useState(-1);
+  const [updateTileCheck, setUpdateTileCheck] = useState(true);
```

Comment:
should this be updated to updatedTilingCheck, setUpdatedTilingCheck to be consistent with the other

Reply:
also what's the difference between updateTileCheck and tilingCheck?